dScripts/02_server/Map/AM/AmSkullkinTower.cpp

```diff
@@ -144,13 +144,10 @@ void AmSkullkinTower::OnChildRemoved(Entity* self, Entity* child) {
 			);
 
 			for (const auto& mission : missions) {
-				int32_t missionID = 0;
+				const auto missionID = GeneralUtils::TryParse<LOT>(mission);
```

Comment:
missions arent lots, should be changed to int32_t.  a LOT is a LEGO Object Template aka a pre-fab object to be placed in the world.